docs/getting-started/development.md

```diff
@@ -0,0 +1,39 @@
+# Development
```

Comment:
Does this file overlap with the traditional `contributing.md` file?

Reply:
Not really, but I should link that doc in `contributing.md`

Reply:
I feel like we should move the relevant bits of this over to contributing, and delete this file.